test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1419,6 +1419,16 @@ def forward(self, x: float, y: float):
         y = 2
         self.run_test(ArithmeticModule(), (x, y))
 
+    def test_tuple(self):
+        class Tuple(torch.nn.Module):
+            def forward(self, x):
+                l = (x, None, (x, None))
+                return (x, l)
+
+        model = Tuple()
+        x = torch.randn(3, 4)
+        self.run_model_test_with_external_data(Tuple(), (x,), example_outputs=model(x))
```

Comment:
why `run_model_test_with_external_data` instead of the usual `run_test`?

Reply:
Fixed